src/express-middleware/portrait.js

```diff
@@ -0,0 +1,28 @@
+import { getAPIResourceWithAuth } from '@plone/volto/helpers';
+
+const HEADERS = ['content-type', 'content-disposition', 'cache-control'];
+
+function portraitMiddleware(req, res, next) {
+  getAPIResourceWithAuth(req)
+    .then((resource) => {
+      // Just forward the headers that we need
+      HEADERS.forEach((header) => {
+        if (resource.headers[header]) {
+          res.set(header, resource.headers[header]);
+        }
+      });
+      res.send(resource.body);
+    })
+    .catch(next);
+}
```

Comment:
@instification Would it make sense to reuse imageMiddleware here instead of duplicating the code?